Makefile

```
@@ -120,6 +120,11 @@ sbom-docker-image:
 attest-docker-image:
 	@cosign attest --type slsaprovenance --predicate predicate.json $(DOCKER_IMAGE):$(VERSION)
 
+.PHONY: update-helm-docs
+update-helm-docs:
+	GO111MODULE=on go install github.com/norwoodj/helm-docs/cmd/helm-docs@latest
```

Suggestion: we already export `GO111MODULE` https://github.com/jetstack/jetstack-secure/pull/408/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L37
```suggestion
	go install github.com/norwoodj/helm-docs/cmd/helm-docs@latest
```

---

Suggestion(non-blocking): can we set a specific version of the tool?